### DISCUSSION TITLE
Whether arbitrary free connections like progressOn can be supported

### DISCUSSION THREAD

**Discussion by zhongyao-x** [bcakmakoglu/vue-flow#1584] [Q&A]
Like this
![微信截图_20240819094106](https://github.com/user-attachments/assets/c761c63d-ea54-46e0-8586-fb72bea49789)

**Comment by zhongyao-x** [comment]
Any number of connections at any location

**Comment by bcakmakoglu** [comment]
You might be looking for examples like these:
https://github.com/bcakmakoglu/vue-flow/tree/master/examples/vite/src/EasyConnect
https://github.com/bcakmakoglu/vue-flow/tree/master/examples/vite/src/FloatingEdges

You won't find these on the docs though but you can copy them out of the repo from the links above ^^
